Created by Jasper Grimm (UoY)
Ensure failing tests results in a non-zero exit code
diff -Nru vt-0.57721d/test/test.sh vt-0.57721e/test/test.sh
--- vt-0.57721d/test/test.sh	2021-05-17 16:42:26.000000000 +0100
+++ vt-0.57721e/test/test.sh	2021-05-17 17:00:48.000000000 +0100
@@ -234,4 +234,7 @@
 echo -n " "
 echo -n ${PASSED_TESTS}  
 echo -n " / "
-echo ${NO_TESTS}  
+echo ${NO_TESTS}
+if [ ${PASSED_TESTS} -ne  ${NO_TESTS} ]; then
+    exit 1
+fi
